onnx/defs/tensor/defs.cc

```diff
@@ -23,12 +24,15 @@ NOTE: Casting to and from strings is not supported yet.
           "The data type to which the elements of the input tensor are cast."
           "Strictly must be one of the types from DataType enum in TensorProto",
           AttrType::STRING)
-    .Input(0, "input", "Input tensor to be cast.")
+    .Input(0, "input", "Input tensor to be cast.", "T")
     .Output(
         0,
         "output",
         "Output tensor with the same shape as input with type "
-        "specified by the 'to' argument");
+        "specified by the 'to' argument",
+        "T")
```

Comment:
The output doesn't have to be the same type as the input. It should be type T2, a different type variable. I assume that "to" is an attribute that specifies the type T2?

Reply:
good catch! fixed. Thanks!